content/blog/2019-07-09-winter-school.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Applications Now Open for the Urssi Winter School in Research Software Engineering"
+date: 2019-07-09T14:13:01-07:00
+author: "Kyle Niemeyer, Jeffrey Carver, Karthik Ram"
+---
+
+
+__Do you develop software for your research? Do you have some basic skills but desire more?__ If so, you might be interested in the URSSI Winter School in Research Software Engineering. As part of the URSSI institute planning, we are planning a pilot 2.5-day workshop on research software engineering skills. This is aimed at early-career researchers, including graduate students and postdocs, who are familiar with the basics such as the Unix shell, version control with Git, and Python programming, and would like to learn more about best-practices for developing research software.
+
+<span style="text-decoration:underline;">Target Audience</span>
+
+The ideal candidate to attend this Winter School is a graduate student or postdoc who understands the basics of programming and version control but wants to learn more about software development practices. To get the most benefit from this workshop, we expect students to be familiar with the unix shell, Python and Git at the level taught at a Software Carpentry workshop.
+
+<span style="text-decoration:underline;">Format and topics</span>
+
+This 2.5 day, hands-on workshop will expose participants to a series of topics necessary for developing sustainable research software. Topics to be covered include:
+
+
+
+*   **Software design and modularity**
+*   **Collaborative Software Development via GitHub**
```

Comment:
I would just make this `*   **Collaborative software development via GitHub**` so the capitalization matches the rest

Reply:
fixed